ckcp/openshift_dev_setup.sh

```diff
@@ -254,8 +258,9 @@ patches:
   echo -n "  - Workloadcluster pipeline-cluster registration: "
   if ! KUBECONFIG="$KUBECONFIG_KCP" kubectl get WorkloadCluster local >/dev/null 2>&1; then
     (
+      KCP_VERSION="$(yq -e '.images[0].newTag' "$ckcp_dev_dir"/kustomization.yaml)"
       KUBECONFIG="$KUBECONFIG_KCP" kubectl kcp workload sync local \
-      --syncer-image ghcr.io/kcp-dev/kcp/syncer:release-0.4 \
+      --syncer-image ghcr.io/kcp-dev/kcp/syncer:$KCP_VERSION \
       --resources conditions.tekton.dev,pipelines.tekton.dev,pipelineruns.tekton.dev,pipelineresources.tekton.dev,runs.tekton.dev,tasks.tekton.dev,taskruns.tekton.dev > "$kube_dir/syncer.yaml"
```

Comment:
As discussed, we must also add tekton triggers resources here to sync without which ./test.sh fails for triggers examples.

Reply:
`+ echo 'Arg triggers passed. Running triggers tests...'
Arg triggers passed. Running triggers tests...
+ echo 'Simulating a Github PR through a curl request which creates a TaskRun (from tektoncd/triggers/examples)'
Simulating a Github PR through a curl request which creates a TaskRun (from tektoncd/triggers/examples)
+ KUBECONFIG=/home/bnr/workspace/pipelines-service/ckcp/work/kubeconfig/admin.kubeconfig
+ kubectl apply -f https://raw.githubusercontent.com/tektoncd/triggers/v0.18.0/examples/v1beta1/github/github-eventlistener-interceptor.yaml
unable to recognize "https://raw.githubusercontent.com/tektoncd/triggers/v0.18.0/examples/v1beta1/github/github-eventlistener-interceptor.yaml": no matches for kind "EventListener" in version "triggers.tekton.dev/v1beta1"
unable to recognize "https://raw.githubusercontent.com/tektoncd/triggers/v0.18.0/examples/v1beta1/github/github-eventlistener-interceptor.yaml": no matches for kind "TriggerBinding" in version "triggers.tekton.dev/v1beta1"
unable to recognize "https://raw.githubusercontent.com/tektoncd/triggers/v0.18.0/examples/v1beta1/github/github-eventlistener-interceptor.yaml": no matches for kind "TriggerTemplate" in version "triggers.tekton.dev/v1beta1"
`

Reply:
As discussed that we remove the trigger resources to the way it was before (just the pipeline resources)